nixos/services/ceph/client.nix

```diff
@@ -166,6 +190,7 @@ in
       }
     ];
 
+    flyingcircus.services.ceph.fc-ceph.enable = true;
```

Comment:
Why do ceph clients (e.g. backyservers, KVM hosts) require fc-ceph? My understanding so far was that fc-ceph is intended as a util to manage ceph cluster **hosts**. Right now this is limited by the intertwining of fc-ceph and fc-luks, but there is at least the *intention* to factor these two out.